### PR TITLE
Activate emscripten with no-embedded option.

### PR DESCRIPTION
**Pull request by madmiraal**
Javascript's Travis CI is failing because javascript's `can_build()` is returning `false`:
https://github.com/godotengine/godot/blob/0f1da724924fc494bfd316c57757e0e1d7b90143/platform/javascript/detect.py#L14-L15

Emscripten is moving to an embedded config model:
https://github.com/emscripten-core/emscripten/issues/9543
The activate option was changed to --embedded by default:
https://github.com/emscripten-core/emsdk/pull/472
So this is a quick fix to mirror their own quick fix:
https://github.com/emscripten-core/emscripten/pull/11293

Basically, we need to use --no-embedded until we can update our own check for a properly installed Emscripten.

### CODE REVIEW

**Review by akien-mga — APPROVED** [review]

**Comment by akien-mga** [comment]
Thanks!

**Comment by akien-mga** [comment]
Removing `cherrypick:3.2` label as it seems we don't have javascript builds on Travis there.

**Comment by akien-mga** [comment]
This is no longer valid, latest emscripten removed the `no-embedded` option so CI builds are failing again.

CC @Faless
